ga4gh/client.py

```diff
@@ -98,6 +98,24 @@ def _runGetRequest(self, objectName, protocolResponseClass, id_):
         """
         raise NotImplemented()
 
+    def getBioSample(self, bioSampleId):
+        """
+        Perform a get request for the given BioSample.
+        :param bioSampleId:
+        :return: protocol.BioSample
+        """
+        return self._runGetRequest(
+            "biosamples", protocol.BioSample, bioSampleId)
+
+    def getIndividual(self, individualId):
+        """
+        Perform a get request for the given Individual.
+        :param individualId:
+        :return: protocol.Individual
```

Comment:
Likewise here. This is user-facing documentation, so we should keep it tidy and consistent.